.changeset/shiny-ads-fix.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/admin-theme": major
+"@comet/cms-admin": major
+"@comet/admin": major
+---
+
+Bump @mui/x-data-grid from v5 to v6, which has breaking changes in DataGrid.
+Change public API of `useDataGridRemote` hook.
+Change public API from `muiGridSortToGql` function.
```

Comment:
Should we add the changes to the public APIs into the changeset?

Reply:
you mean, the explicit change what i mentioned in the changelog?

Reply:
Yes. You've mentioned them in the migration guide, but not in the changelog (=changeset).

Reply:
we can do so 👍

Reply:
done